modules/tinymce/src/plugins/directionality/test/ts/browser/DirectionalitySanityTest.ts

```diff
@@ -47,4 +47,139 @@ describe('browser.tinymce.plugins.directionality.DirectionalitySanityTest', () =
       });
     }));
   });
+
+  it('should set two paragraphs to rtl and ltl', () => {
```

Comment:
Can you please add the JIRA number as a prefix here? We always try to include the JIRA number so we can trace why a test was added. eg:
```suggestion
  it('TINY-4589: should set two paragraphs to rtl and ltl', () => {
```